structure_tests/structure_test.go

```diff
@@ -34,59 +35,59 @@ func TestRunCommand(t *testing.T) {
 		stdout := outbuf.String()
 		stderr := errbuf.String()
 
-		for _, errStr := range tt.ExpectedError {
+		for _, errStr := range tt.GetExpectedError() {
```

Comment:
i'm a little confused what you mean here. are you talking about just having a generic test driver method that's agnostic to the type of test it's running? this was something i had tried to figure out, but didn't have much success because each of the test types have specific "checks" embedded into them (e.g. expected error in a command test, or whether or not a file should exist in the existence tests).